testssl.sh

```diff
@@ -3197,36 +3195,43 @@ run_security_headers() {
      fi
 
      pr_bold " Security headers             "
-     for header in $good_header; do
-          [[ "$DEBUG" -ge 5 ]] &&  echo "testing \"$header\""
+     # X-XSS-Protection is useless and at worst harmful, see https://news.ycombinator.com/item?id=20472947
+     for header_and_svrty in "X-Frame-Options OK" \
+                             "X-Content-Type-Options OK" \
+                             "Content-Security-Policy OK" \
+                             "X-Content-Security-Policy OK" \
+                             "X-WebKit-CSP OK" \
+                             "Content-Security-Policy-Report-Only OK" \
+                             "Expect-CT OK" \
+                             "Permissions-Policy OK" \
+                             "X-XSS-Protection LOW" \
```

Comment:
I would not label it in any way. ``0`` was bad maybe a couple of years back, ``1`` was the default in some browsers ``1; mode=block`` was where some people frowned upon. This was a lack of testssl.sh not to look at the values.

Moving forward now I think it should just be neutral or to emphasize in a good manner. "A good manner" though hasn't been determined yet.

Reply:
I assume "neutral" == "INFO". This will (as it was before for the ```other_headers```) output the header with a "neutral" color, which is currently ```pr_litecyan```.
There is a TODO comment for the lack of looking at the values. However, evaluating the values is unique per header and I agree with opening a new pull request.